src/ui/pages/Security/AutoLockTimer.tsx

```diff
@@ -41,49 +46,74 @@ export const AUTO_LOCK_TIMER_OPTIONS_TITLES = Object.fromEntries(
 export function AutoLockTimer() {
   const { globalPreferences, setGlobalPreferences } = useGlobalPreferences();
   const value = globalPreferences?.autoLockTimeout;
+  const [selectedValue, setSelectedValue] = useState<
+    GlobalPreferences['autoLockTimeout'] | null
+  >(null);
+
+  const isNewValue = Boolean(selectedValue && selectedValue !== value);
 
   return (
-    <PageColumn>
-      <NavigationTitle title="Auto-Lock Timer" />
-      <PageTop />
-      <VStack gap={8}>
-        <SurfaceList
-          items={AUTO_LOCK_TIMER_OPTIONS.map((preference, index) => ({
-            key: index,
-            isInteractive: true,
-            pad: false,
-            component: (
-              <SurfaceItemLabel>
-                <UIText
-                  kind="body/regular"
-                  color={
-                    value === preference.value ? 'var(--primary)' : undefined
-                  }
+    <Background backgroundKind="white">
```

Comment:
I consider `<Background />` deprecated, use `useBackgroundKind(...)` instead. It does the same things but doesn't affect indentation